deploy/olm-catalog/akka-cluster-operator/0.2.1/akka-cluster-operator-certified.v0.2.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v0.2.1
```

Comment:
is it ok to have version in the name

Reply:
Since the CSV pertains to the specific version, the naming scheme is the name of the Operator + the semantic version number, [eg mongodboperator.v0.3.](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#csv-metadata)